docs/man/Dockerfile.5.md

```diff
@@ -69,7 +69,7 @@ A Dockerfile is similar to a Makefile.
   multiple images. Make a note of the last image ID output by the commit before
   each new **FROM** command.
 
-  -- If no tag is given to the **FROM** instruction, latest is assumed. If the
+  -- If no tag is given to the **FROM** instruction, the latest is assumed. If the
```

Comment:
But There is not much diffrence between the latest and the 'latest' .
Also, I think it is not the sentence which needs to be emphasized.
so that I think the latest is more reasonable.